regnet/examples/syntax_edge_cases.json

```diff
@@ -25,16 +27,19 @@
     "edges": [
       {
         "id": "rabbit_growth",
+        "name": "Rabbit growth",
+        "description": "Natural growth rate of a rabbit population",
         "source": "R",
         "target": "R",
         "sign": true,
         "properties": {
-          "name": "Rabbit growth",
```

Comment:
This is incorrect, `name` and `description` should be in the `properties` of the edge because it provides no structural meaning to the edge. The `vertices` do not have a nested `properties` field because there are no real structure of a and so all keys under it are actually just a `properties` table basically.

This does expose issues in our JSON validation because this should not validate. I will look into this soon to resolve this bug in our JSON Schema

Reply:
I realized later in the review that our original JSON Schema was correct and changes were made to it to make this validate. The 2nd half of this comment can be ignored as it was brought back up in a separate conversation here.

Reply:
I think we could follow what @mehalter is saying here i.e., name and description could remain inside properties. I don't have a strong opinion either way though.

Reply:
I think I understand what you mean. I've moved `name` and `description` into `properties` of both RegNet edges and PetriNet transitions.